requirements/requirements.txt

```diff
@@ -388,7 +388,9 @@ twisted[tls]==20.3.0
     #   daphne
 txaio==20.12.1
     # via autobahn
-urllib3==1.25.8
+typing-extensions==3.10.0.2
+    # via aiohttp
+urllib3==1.26.5
```

Comment:
https://github.com/urllib3/urllib3/blob/main/CHANGES.rst#1260-2020-11-10

Only 1 changelog entry from 1.25.x -> 1.26.x that looks scary. I guess kubernetes or requests requires this bump?

```
Deprecated negotiating TLSv1 and TLSv1.1 by default. Users that still wish to use TLS earlier than 1.2 without a deprecation warning should opt-in explicitly by setting ssl_version=ssl.PROTOCOL_TLSv1_1 (Pull #2002) Starting in urllib3 v2.0: Connections that receive a ``DeprecationWarning`` will fail
```

Reply:
https://github.com/advisories/GHSA-q2q7-5pp4-w6pg